dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -745,6 +746,16 @@ open class Jpql : JpqlDsl {
         return Expressions.currentTime()
     }
 
+    /**
+     * Creates an expression that represents the current timestamp(datetime).
```

Comment:
The datetime and timestamp may be different in different contexts. For example, to a MySQL user, datetime and timestamp will look like separate types. 

I don't think the Kotlin JDSL needs to interpret them separately and guide the user to datetime.

Reply:
I didn't find any other issues, so I'll fix this.

Reply:
Oh! Thanks, I also noticed that timestamp didn't seem to include time, so you're right!
Thanks for letting me know, that's helpful haha